qa/smoke-test-http/src/javaRestTest/java/org/elasticsearch/http/BlockedSearcherRestCancellationTestCase.java

```diff
@@ -71,11 +73,32 @@ protected boolean addMockInternalEngine() {
         return false;
     }
 
+    // add opaque id to http request header if it does not exist
+    // returns new or existing id
+    private static String setOpaqueIdIfNotExists(Request request) {
+        String opaqueId;
+        if (request.getOptions().containsHeader(Task.X_OPAQUE_ID_HTTP_HEADER)) {
```

Comment:
I think this is dead code today since none of the requests include an opaque-id header. Let's just assert it's absent.